### ISSUE TITLE
Unable to connect to access point when in monitor mode 

### ISSUE THREAD

**Issue by LeonKarabchevsky** [seemoo-lab/nexmon#317]
I'm using Raspberry PI 3B+ and I have successfully started monitor mode on mon0 by using:
```
iw phy `iw dev wlan0 info | gawk '/wiphy/ {printf "phy" $2}'` interface add mon0 type monitor
ifconfig mon0 up
```

The documentation states that:  "It is possible to connect to an access point or run your own access point in parallel to the monitor mode interface on the wlan0 interface"
But I was unable to do so, as using `iwlist wlan0 scan` returns nothing 

If I use `nexutil -m0` the scan returns available access points (but the monitor mode stops working )

**Comment by matthiasseemoo** [comment]
Good point, monitor mode on the bcm43455c0 is implemented a bit differently
than on other devices, as we do not create a copy of the incoming frame in
the wl_monitor function. But I have an idea on how to make it work, just
need ro find some time to try it.

LeonKarabchevsky <notifications@github.com> schrieb am Mo., 5. Aug. 2019,
21:46:
